test/compilable/test19728.d

```diff
@@ -0,0 +1,50 @@
+/*
+TEST_OUTPUT:
+---
+tuple((A), (B))
+tuple((A), (B), 0)
+tuple((A), (B), (A))
+tuple((A), (B), (A), (B))
+tuple((A), (B), (A), (B))
+tuple((A), (B), (A), (B), (A), (B), (A), (B))
+tuple((Attr))
+---
+*/
+
+// Issue 19728
+enum A; enum B; enum Dummy = 0;
+
+alias Seq(T...) = T;
+
+@Seq!(A,B) struct Foo1 {}
+@Seq!(A,B, Dummy) struct Foo2 {}
+@Seq!(A,B,A) struct Foo3 {}
+@Seq!(Seq!(A,B,A,B)) struct Foo4 {}
+@Seq!(A,Seq!(B,A),B) struct Foo5 {}
+@Seq!(A,Seq!(B,A),B) @Seq!(A,B,A,B) struct Foo6 {}
+
+pragma(msg, __traits(getAttributes, Foo1));
+pragma(msg, __traits(getAttributes, Foo2));
+pragma(msg, __traits(getAttributes, Foo3));
+pragma(msg, __traits(getAttributes, Foo4));
+pragma(msg, __traits(getAttributes, Foo5));
+pragma(msg, __traits(getAttributes, Foo6));
+static assert(is(typeof(__traits(getAttributes, Foo1))));
```

Comment:
There should be tests to verify what the UDAs actually are.

Reply:
Is better now?

Reply:
Yes, thanks.